src/cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -2967,68 +2973,72 @@ msgstr ""
 "Benachrichtigungen verwalten."
 
 #: views/push_notifications/push_notification_view.py:84
-msgid "Push notification saved successfully."
-msgstr "Push-Benachrichtigung wurde erfolgreich gespeichert."
+msgid "Push notification was successfully saved"
+msgstr "Push-Benachrichtigung wurde erfolgreich gespeichert"
 
 #: views/push_notifications/push_notification_view.py:87
-msgid "Push notification created successfully."
-msgstr "Push-Benachrichtigung wurde erfolgreich erstellt."
+msgid "Push notification was successfully created"
+msgstr "Push-Benachrichtigung wurde erfolgreich erstellt"
 
 #: views/push_notifications/push_notification_view.py:144
-msgid "Push notification was successfully sent."
-msgstr "Push-Benachrichtigung wurde erfolgreich gespeichert."
+msgid "Push notification was successfully sent"
+msgstr "Push-Benachrichtigung wurde erfolgreich gesendet"
 
 #: views/push_notifications/push_notification_view.py:148
 msgid "Error occurred sending the push notification"
-msgstr "Push-Benachrichtigung konnte nicht gesendet werden"
+msgstr "Fehler während des Sendens der Push-Benachrichtigung"
 
 #: views/regions/region_actions.py:22
-msgid "Region was successfully deleted."
-msgstr "Region wurde erfolgreich gelöscht."
+msgid "Region was successfully deleted"
+msgstr "Region wurde erfolgreich gelöscht"
 
 #: views/regions/region_view.py:54
-msgid "Region saved successfully."
-msgstr "Region wurde erfolgreich gespeichert."
+msgid "Region was saved successfully"
+msgstr "Region wurde erfolgreich gespeichert"
 
 #: views/regions/region_view.py:56
-msgid "Region created successfully"
-msgstr "Region wurde erfolgreich erstellt."
+msgid "Region was created successfully"
+msgstr "Region wurde erfolgreich erstellt"
 
 #: views/roles/role_view.py:37
-msgid "Role saved successfully"
-msgstr "Rolle wurde erfolgreich gespeichert."
+msgid "Role was successfully saved"
+msgstr "Benutzer wurde erfolgreich gespeichert"
```

Comment:
```suggestion
msgstr "Rolle wurde erfolgreich gespeichert"
```

Reply:
And just 2 lines later the same again.

Reply:
I guess we need a translation clean up and validation task at some point.